smoke/publish-local-registry.sh

```
@@ -5,7 +5,7 @@ progName=$0
 projectRoot=$(pwd)
 cd $(dirname $progName)
 
-if [ $(which podman) -a "$FP_CI" != "fp_ci" ]
+if [ "$(which podman)" -a "$FP_CI" != "fp_ci" ]
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)